pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -551,20 +571,21 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 
 	if hasWindowsNodes {
 		fluentdCfg = &render.FluentdConfiguration{
-			LogCollector:    instance,
-			ESSecrets:       esSecrets,
-			ESClusterConfig: esClusterConfig,
-			S3Credential:    s3Credential,
-			SplkCredential:  splunkCredential,
-			Filters:         filters,
-			EKSConfig:       eksConfig,
-			PullSecrets:     pullSecrets,
-			Installation:    installation,
-			ClusterDomain:   r.clusterDomain,
-			OSType:          rmeta.OSTypeWindows,
-			TrustedBundle:   trustedBundle,
-			ManagedCluster:  managedCluster,
-			UsePSP:          r.usePSP,
+			LogCollector:       instance,
+			ESSecrets:          esSecrets,
+			ESClusterConfig:    esClusterConfig,
+			S3Credential:       s3Credential,
+			SplkCredential:     splunkCredential,
+			Filters:            filters,
+			EKSConfig:          eksConfig,
+			PullSecrets:        pullSecrets,
+			Installation:       installation,
+			ClusterDomain:      r.clusterDomain,
+			OSType:             rmeta.OSTypeWindows,
+			TrustedBundle:      trustedBundle,
+			ManagedCluster:     managedCluster,
+			UsePSP:             r.usePSP,
+			UseUserCertificate: useUserCertificate,
```

Comment:
Maybe the variable name could include a hint that this is for syslog. Like `userSyslogCertificate` perhaps?

Reply:
updated